examples/servients/opcua-cli/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@node-wot/opcua-cli",
-  "version": "0.7.2",
+  "version": "0.7.6",
```

Comment:
We are about to move to "0.7.7" ... can you update this as well. Thanks!

Reply:
ok, sure